src/app/event/event-detail-controller.js

```diff
@@ -1,7 +1,9 @@
 angular.module('songster.event')
 
-    .controller('EventDetailController', function EventCtrl($scope,$http, $location, $event, $state, $rootScope) {
+    .controller('EventDetailController', function EventCtrl($scope,$http,$account, $location, $event, $state, $rootScope) {
         $scope.event = $event.getEvent();
+        $scope.user = $account.getUser();
+        $scope.isDJ =  $account.getUser() !== null && $event.getEvent().owner_id ==  $account.getUser()._id;//$rootScope.isDj();
```

Comment:
$account.getUser() !== null could be replaced by $account.getUser()

see: https://dorey.github.io/JavaScript-Equality-Table/